### PR TITLE
Support multiple maps in training

### LINKED ISSUE #72: Support multiple maps during training,

### PR DESCRIPTION
**Pull request by xluox**
Resolve #72 
Swapping maps in the vec_evn file every time a match is finished. Evaluation will not trigger swapping maps.

### CODE REVIEW

**Review by vwxyzjn — CHANGES_REQUESTED** [review]

File: gym_microrts/envs/vec_env.py [hunk 1]
```diff
@@ -12,6 +13,14 @@
 
 import gym_microrts
 
+MULTIPLE_MAP_PATHS = [
+    "/home/yifuli/Documents/gym-microrts/gym_microrts/microrts/maps/16x16/basesWorkers16x16A.xml",
+    "/home/yifuli/Documents/gym-microrts/gym_microrts/microrts/maps/16x16/basesWorkers16x16B.xml",
+    "/home/yifuli/Documents/gym-microrts/gym_microrts/microrts/maps/16x16/basesWorkers16x16C.xml",
```
[the remark this answers is not in the document]
Comment (xluox) [reply gym_microrts/envs/vec_env.py:19]: Not sure how to add these codes.

[the remark this answers is not in the document]
Comment (vwxyzjn) [reply gym_microrts/envs/vec_env.py:19]: Try print out `os.path.join(gym_microrts.__path__[0], "microrts")` see if it helps.

[the remark this answers is not in the document]
Comment (vwxyzjn) [reply gym_microrts/envs/vec_env.py:19]:
```
>>> import gym_microrts
>>> import os
>>> print(os.path.join(gym_microrts.__path__[0], "microrts"))
/home/costa/Documents/go/src/github.com/gym-microrts/gym_microrts/microrts
```

[the remark this answers is not in the document]
Comment (xluox) [reply gym_microrts/envs/vec_env.py:19]: Got it

**Review by vwxyzjn — APPROVED** [review]
This makes sense to me. Thank you! Feel free to click the merge button :)

**Pull request merged**